static/styles.js

```diff
@@ -1,212 +1,183 @@
-    chat.style.color = "#ffffff";
-    message.style.color = "#000000";
-    chatbox.style.backgroundColor = "#181616";
-    sides.style.backgroundColor = "#121212";
-    sidebar.style.backgroundColor = "#171717";
-    topleft.style.backgroundColor = "#f5f2f2";
-    topleft.style.color = "#192080";
-    send.style.backgroundColor = "#192080";
-    send.style.color = "#ffffff";
-    sidenav.style.backgroundColor = "#111";
-    // for loop to cycle thru links in sidebar
-    for (let i = 0; i < snav_iter; i++) {
-        snav_text[i].style.color = "#818181";
-    }
+    setStyles(...themes[themeName]);
 }
 
-function setLightstyle() {
-    // get all id tags
-    let body = document.getElementsByTagName("body")[0]; 
-    let chat = document.getElementById("chat");
-    let message = document.getElementById("message");
-    let chatbox = document.getElementById("chatbox");
-    let sides = document.getElementById("sides");
-    let topleft = document.getElementById("topleft");
-    let send = document.getElementById("send");
-    let sidebar = document.getElementById("activenav");
-    let sidenav = document.getElementsByClassName("sidenav")[0];
-    let snav_text = sidenav.getElementsByTagName("a");
-    let snav_iter = snav_text.length;
-    // then set the color to what it is in the css document
-    body.style.backgroundColor = "#c0bfbc";
-    chat.style.color = "#000000";
-    message.style.color = "#000000";
-    chatbox.style.backgroundColor = "#deddda";
-    sides.style.backgroundColor = "#c0bfbc";
-    sidebar.style.backgroundColor = "#000000";
-    topleft.style.backgroundColor = "#5A5A5A";
-    topleft.style.color = "#1b0670";
-    send.style.backgroundColor = "#3daec4";
-    send.style.color = "#33575e";
-    sidenav.style.backgroundColor = "#b9c6c9";
-    // for loop to cycle thru links in sidebar
-    for (let i = 0; i < snav_iter; i++) {
-        snav_text[i].style.color = "#192080";
-    }
-}
+const darkTheme = [
+    "#000000",
+    "#ffffff",
+    "#000000",
+    "#181616",
+    "#121212",
+    "#171717",
+    "#f5f2f2",   // Sidebar Background Color
+    "#3b3b3b",   // Sidebar Border Color
+    "#c73228",   // Sidebar Text Color
+    "#192080",
+    "#192080",
+    "#ffffff",
+    "#111",
+    "#818181",
+];
 
+const lightTheme = [
+    "#c0bfbc",   // Background Color
+    "#000000",   // Chat Color
+    "#000000",   // Message Color
+    "#deddda",   // Chatbox Color
+    "#c0bfbc",   // Sides Color
+    "#000000",   // Sidebar Background Color
+    "#3b3b3b",   // Sidebar Border Color
+    "#ffffff",   // Sidebar Text Color
+    "#5A5A5A",   // Topleft Background Color
+    "#1b0670",   // Topleft Text Color
+    "#3daec4",   // Send Button Background Color
+    "#33575e",   // Send Button Text Color
+    "#b9c6c9",   // Sidenav Color
+    "#192080",   // Sidenav Link Color
+    "#000000",     // Room Text color
+    "#000000",      // Room Line color
+    "#fff"       //Topbar Background Color
```

Comment:
expand out to `#ffffff`, or in this case, maybe change to `black`

Reply:
So I expanded it out to ```#ffffff```, because it blends in with the rest of the code better than ```black```